manifests/pulp.pp

```diff
@@ -129,6 +142,39 @@
     }
   }
 
+  if $expose_mongodb {
+    include certs::mongodb
+    include certs::mongodb_client
+
+    Class['certs::mongodb_client'] -> Class['pulp']
+
+    $mongodb_bind_ip = $expose_mongodb_ipv6 ? {
+      true  => ['0.0.0.0', '::/0'],
+      false => ['0.0.0.0'],
+    }
```

Comment:
```suggestion
    }

    $mongodb_ipv6 = $expose_mongodb_ipv6 ? {
      true  => true,
      false => undef,
    }
```

I think this would work.

Reply:
Why is this needed?

Reply:
It's not. I like your solution better.